packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.test.tsx

```diff
@@ -8,6 +8,8 @@ import { IColumn } from '../DetailsList/DetailsList.types';
 import { List } from '../List/List';
 import { GroupShowAll } from './GroupShowAll';
 import { Link } from '../Link/Link';
+import { GroupHeader } from '.';
```

Comment:
these imports will probably fail the build. Can these be imported from `'./index` or from a physical file? (AMD imports do not automatically resolve index files.)

Reply:
The AMD thing won't be an issue here since it's a test, but this will probably still fail the lint-imports script (if it runs on tests)

Reply:
Thats true; i was just assuming the lint-imports script would have caught this.